dotnet/src/webdriver/ErrorResponse.cs

```diff
@@ -102,38 +86,22 @@ public ErrorResponse(Dictionary<string, object> responseValue)
         /// <summary>
         /// Gets or sets the message from the response
         /// </summary>
-        public string Message
-        {
-            get { return this.message; }
-            set { this.message = value; }
-        }
+        public string? Message { get; } = string.Empty;
```

Comment:
The `Message` property differs from its siblings. I think this property should either become non-nullable or lose its `string.Empty` default value. After the PR changes and before, as I see this property is never set to `null` and logically (according to constructor logic) should not return `null`, so I would make it non-nullable.

Reply:
That's a good point. It's worth noting that the `stackTrace` check doesn't work, because it comes back as a `string` instead of a list of stack frames, and there's an extra `error` entry in the scenarios I tested this against.

But I guess that's a separate issue